spec/reports/cost_report_spec.rb

```diff
@@ -6,33 +6,37 @@
 require "clustering/cluster_ht_item"
 require "data_sources/ht_organizations"
 
-RSpec.xdescribe Reports::CostReport do
+
+RSpec.describe Reports::CostReport do
   include_context "with tables for holdings"
 
-  let(:alo) { "allow" }
-  let(:dni) { "deny" }
+  let(:allow) { "allow" }
```

Comment:
@mwarin I don't recall what specifically motivated pulling these out. If we still feel there's value in having them be `let`s rather than having them inlined as strings, maybe this is a reasonable compromise?